src/lambda/commands/showStackTrace.ts

```diff
@@ -0,0 +1,34 @@
+/*!
+ * Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+import _ = require('lodash')
+import * as vscode from 'vscode'
+import { BaseTemplates } from '../../shared/templates/baseTemplates'
+import { ErrorNode } from '../explorer/errorNode'
+import { ErrorTemplates } from '../templates/errorTemplates'
+
+export async function showStackTrace(element: ErrorNode) {
+    try {
+
+        const view = vscode.window.createWebviewPanel(
+            'html',
+            `Error information for ${element.parent.label}`,
+            -1
+        )
+
+        const baseTemplateFn = _.template(BaseTemplates.SIMPLE_HTML)
+        view.webview.html = baseTemplateFn({ content: '<h1>Loading...</h1>' })
+
+        const getConfigTemplateFn = _.template(ErrorTemplates.ERROR_INFORMATION)
+        view.webview.html = baseTemplateFn({
+            content: getConfigTemplateFn(element)
+        })
+    } catch (err) {
```

Comment:
I checked the `LambdaPolicyView`. It's very well written and I believe that we can create an abstract class to handle all these kind of views with the general methods (`createView`, `dispose`, `load`, `showContent`,  `showError`, `showErrorContent` and `showLoadingContent`) and address these change in a separated PR. 

But I also saw that there is the issue #146, so I think that the above PR will not be useful at all.

What could I do for now? Could I create a small class alike `LambdaPolicyView` inside `showStackTrace.ts` just to address this UI update issue in case of error?

Reply:
#146 is outdated. If you're interested in making a reusable class as mentioned in a separate PR, that sounds great. I think for this PR, the catch can replace the webview text with the error message as a simple way of wrapping up this PR.